pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -740,6 +762,11 @@ public static ViewFilter parseFilteringQuery(String str, ViewFilter viewFilter)
                             } else if (word.equals(Filter.LIKE)) {
                                 state = 8;
                                 break;
+                            } else if (word.equals(Filter.IS)) {
+                                exact = false;
+                                range = false;
+                                state = 5;
```

Comment:
what does `state=5`  mean?  please add a comment explaining

Reply:
as comment for state 5 says: `//if query contains "IS NOT sth"`

Reply:
where is that comment?  sorry, this is a new part of the codebase for me